frontend/src/pages/modelServing/screens/projects/InferenceServiceModal/DataConnectionFolderPathField.tsx

```diff
@@ -9,19 +10,47 @@ type DataConnectionFolderPathFieldProps = {
 const DataConnectionFolderPathField: React.FC<DataConnectionFolderPathFieldProps> = ({
   folderPath,
   setFolderPath,
-}) => (
-  <FormGroup fieldId="folder-path" label="Folder path">
-    <InputGroup>
-      <InputGroupText variant="plain">/</InputGroupText>
-      <TextInput
-        aria-label="folder-path"
-        type="text"
-        value={folderPath}
-        placeholder="eg. data"
-        onChange={(value) => setFolderPath(value)}
-      />
-    </InputGroup>
-  </FormGroup>
-);
+}) => {
+  type validate = 'success' | 'warning' | 'error' | 'default';
```

Comment:
Types are always PascalCase

Also, why do you need this? Infer the data from PF. 

```
React.ComponentProps<typeof FormGroup>['validated']
```

That should get you the value behind the prop of FormGroup's prop `validated`.